src/Destination/S3CmdDestination.php

```diff
@@ -112,7 +113,7 @@ public function all()
             throw new \RuntimeException($process->getErrorOutput());
         }
 
-        return $this->parseS3CmdListOutput($process->getOutput());
+        return new BackupCollection($this->parseS3CmdListOutput($process->getOutput()));
```

Comment:
What is the rationale behind keeping S3CmdDestination after adding flysystem (except for BC, of course)? 
I'm not complaining, just asking :–)

Flysystem seems to be a pretty universal way of pushing the backup to any destination type. Maybe it's worth switching to that engine in some future major release.